examples/gallery/ui/pages/controls_page.slint

```diff
@@ -15,50 +15,57 @@ export component ControlsPage inherits Page {
         title: "Buttons";
 
         HorizontalLayout {
-            spacing: 8px;
             alignment: start;
+            spacing: 8px;
 
             Button {
                 text: "Regular Button";
-                enabled: !GallerySettings.widgets-disabled;
+                enabled: GallerySettings.widgets-enabled;
             }
 
             Button {
                 text: "Button with Icon";
                 icon: @image-url("../../thumbsup.png");
-                enabled: !GallerySettings.widgets-disabled;
+                enabled: GallerySettings.widgets-enabled;
             }
 
             Button {
                 checkable: true;
                 text: self.checked ? "ON" : "OFF";
-                enabled: !GallerySettings.widgets-disabled;
+                enabled: GallerySettings.widgets-enabled;
             }
         }
     }
 
     GroupBox {
-        title: "CheckBox - SpinBox - ComboBox";
+        title: "CheckBox - SpinBox - ComboBox - Switch";
         vertical-stretch: 0;
 
-        HorizontalBox {
+        HorizontalLayout {
             alignment: start;
+            spacing: 8px;
+
             checkbox := CheckBox {
                 text: checkbox.checked ? "(checked)" : "(unchecked)";
                 checked: true;
-                enabled: !GallerySettings.widgets-disabled;
+                enabled: GallerySettings.widgets-enabled;
             }
 
-
-        SpinBox {
-            vertical-stretch: 0;
-            value: 42;
-            enabled: !GallerySettings.widgets-disabled;
-        }
+            SpinBox {
+                vertical-stretch: 0;
+                value: 42;
+                enabled: GallerySettings.widgets-enabled;
+            }
 
             ComboBox {
                 model: ["Select Something", "From this", "Combobox"];
-                enabled: !GallerySettings.widgets-disabled;
+                enabled: GallerySettings.widgets-enabled;
+            }
+
+            Switch {
+                text: self.checked ? "(On)" : "(Off)";
```

Comment:
Same here regarding the text: I don't think the switch should change its text. I think that's an anti-pattern, unless I'm missing something?

```suggestion
                text: "Flight Mode";
```